variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/VariantClassifierTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.core.models;
+
+import org.junit.Rule;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import org.junit.rules.ExpectedException;
+
+public class VariantClassifierTest {
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testSNVs() {
+        assertEquals(VariantType.SNV, VariantClassifier.getVariantClassification("A", "C", 1));
+    }
+
+    @Test
+    public void testDIVs() {
+        assertEquals(VariantType.DEL, VariantClassifier.getVariantClassification("TT", "", 2));
+        assertEquals(VariantType.INS, VariantClassifier.getVariantClassification("", "G", 2));
+        assertEquals(VariantType.INDEL, VariantClassifier.getVariantClassification("A", "GTC", 2));
+        assertEquals(VariantType.INDEL, VariantClassifier.getVariantClassification("AA", "G", 2));
+    }
+    @Test
+    public void testSTRs() {
+        assertEquals(VariantType.TANDEM_REPEAT, VariantClassifier.getVariantClassification("(A)5", "(A)7", 4));
```

Comment:
A short tandem repeat would be also one where the reference is "A" and the alternate is "AAAAA", both displayed explicitly instead of with the syntax "(A)5".